### PR TITLE
PDF - Reassign documents to Laws, Hints, LegalProvision

### LINKED ISSUE #583: No correct determination of different document types in static extract
Since we have now the determination of Law, Hint and LegalProvision working fine in the DataExtract (JSON/XML) we need to transport this to the static extract as well. For the moment only LegalProvision's are recognized correctly. All others are gathered under the Law's in static extract. The Hint section is left blank for the moment.

See page 5/6 of this PDF:
[19b19c96-e7ee-4c24-b9b5-370ccd0427b4.pdf](https://github.com/camptocamp/pyramid_oereb/files/2227877/19b19c96-e7ee-4c24-b9b5-370ccd0427b4.pdf)

I would expect, that document "Wichtige Hinweise" will be delivered under "Weitere Informationen und Hinweise" since it is classified as "Hint" correctly in the corresponding JSON DataExtract:
grafik

### PR DESCRIPTION
**Pull request by ger-benjamin**
GSOREB-232
Fix #583

### CODE REVIEW

**Comment by jwkaltz** [comment]
I am currently trying to extend the test data for the print test case (all documents must contain a document type, now), I will push when I got it working

File: pyramid_oereb/contrib/print_proxy/mapfish_print.py [hunk 1]
```diff
@@ -431,6 +430,26 @@ def _flatten_object(parent, name):
...
+        """
+        uid = Renderer._get_element_of_legal_provision_maybe_uid(document)
+        documentType = document['DocumentType']
+        if documentType == 'LegalProvision':
+            legal_provisions[uid] = document
+        elif documentType == 'Law':
+            laws[uid] = document
+        else:
+            hints[uid] = document
+
```
Comment (pfirpfel) [inline pyramid_oereb/contrib/print_proxy/mapfish_print.py]: Should cover the case if there is no DocumentType and not default to hint.

Comment (jwkaltz) [reply pyramid_oereb/contrib/print_proxy/mapfish_print.py]: if there is no DocumentType, we now get an exception. The default case is only if type is present, but not LegalProvision or Law

**Comment by pfirpfel** [comment]
@jwkaltz  I propose this variant of your "update expected mapfish-print json result"-commit without collateral damage to the unicode string:
https://github.com/camptocamp/pyramid_oereb/commit/8a8fb22c59113c537fe532fcc73d0b930f2a88a9

(exactly the same, but no unecessary replacements; passes tests)

**Comment by vvmruder** [comment]
After all: From point of view this is OK now.

**Pull request merged**